src/level.rs

```diff
@@ -41,6 +41,20 @@ match evt.props().pull::<emit::Level, _>(emit::well_known::KEY_LVL).unwrap_or_de
 ```
 
 The default level is [`Level::Info`].
+
+# Parsing
+
+`Level` has a permissive parser that will match partial and incomplete contents, ignoring case. So long as the start of the text is a submatch of the full level, and any trailing unmatched characters are not ASCII control characters, the level will parse.
+
+For example, the following will all be parsed as `Level::Info`:
+
+- `info`
+- `INFO`
+- `i`
+- `inf`
+- `information`
+- `INFO1`
+- `inf(13)`
```

Comment:
Should call out here that any trailing data is lost so `info3` and `info4` are parsed to the same value.